### PR TITLE
Food module follow up

### PR DESCRIPTION
**Pull request by NPC1314**
## About The Pull Request

Nothing dramatic, food related polishing and minor additions, pie slice rot added, details below

Couple things from the archives, and a addition inspired by Mojave.
* Handpies added (american food in origin but close enough to pierogi to make sense). Add truffles, mince or jacksberry to piedough, fry, enjoy. Dont rot until bitten into and seal broken.
* Truffles now has a sell price as intended (5 mammon)
* Salo can now be put on bread for that real gopnik vibe
* Throwing mince ruins it
* Randomized  eat verbs (sort of split on this but lets try it for a bit)
* Pies don't rot while uncut, slices do however, though they still last a long time, so for actual eternal food preseved/dried/salted/unopened stuff is better.
* Fried potato wedges and sausuage can be combined for meal, also onion added on top
* Some duplicate TG items that keeps annoying me in localtests yeeted, no impact on RT (piedough & sausage)
* Nutrition defines structured a little more so they keep logic 
* Saturation on some food items brought in line with the normal scales and icon names matched up to item names some more
also bread slicing sprite adjusted so the cut is less vertical to make it more interesting and less tiny at the end
* Flour mill sound was placed on the wrong line, only fires when actually milling now.

Spot the difference
foodoss

EDIT
Fixed prezzel bug and cleaned up some rot defines.
Note on rotting - Food got three important traits, longevity, nutrition and mood boost. Costs are resources/time
Part of the module is trying to get a more systematic balance of these things, and combine it with cooking having more soul, less industrial feeling.

TL DR Theres a bunch of foods that never rots, make those if you want that, or you can stick to easy to make low shelflife stuff, to make the options clearer I made a list thats more or less correct

/*	.................   Never spoils   ................... *//*

* Hardtack
* Toast
* Salted fish
* Frybread
* Unbitten handpies
* Biscuit
* Prezzel
* Cheese wheel/wedges
* Salo
* Copiette
* Salumoi
* Uncut pie
* Raw potato, onion, cabbage

/*	.................   Long shelflife   ................... */

* Uncut bread loaf
* Uncut raisin bread
* Uncut cake
* Pastry
* Bun
* Most plated dishes
* Most cooked veggies
* Cooked sausage
* Pie slice
* Bread slice

/*	.................   Decent shelflife   ................... */

* Fresh cheese
* Mixed dishes with meats 
* Fried meats & eggs

/*	.................   Short shelflife   ................... */

* Raw meat
* Berries

/*	.................   Tiny shelflife   ................... */

* Minced meat

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

**Comment by Mojamble** [comment]
WHOOOO!

**Comment by lavillastrangiato** [comment]
Throwing this out there; Is it possible you could add a check for if you're tossing mince onto an adjacent tile that it doesn't get ruined? Throwing it might be needed if you have a massive pile of meat that you can't click on the chest beneath, or some similar issue.

**Comment by BadAtThisGame302** [comment]
Pie slices should remain unable to rot, taking away raisin loaf sure but not pie too.

**Comment by Mojamble** [comment]
> Throwing this out there; Is it possible you could add a check for if you're tossing mince onto an adjacent tile that it doesn't get ruined? Throwing it might be needed if you have a massive pile of meat that you can't click on the chest beneath, or some similar issue.

Alt right click will open up a view menu for sorting that kind of stuff! Can’t recommend it enough.

**Comment by Mojamble** [comment]
> Pie slices should remain unable to rot, taking away raisin loaf sure but not pie too.

Use a crate or invest in salt for dried foodstuffs. Your slashed open slice of pie shouldn’t be able to just sit out forever getting pissed on by rats and going moldy from the fact that we live less than a mile from a swamp.

**Review by Stutternov — APPROVED** [review]
